samples/drivers/display/boards/da1469x_dk_pro.conf

```diff
@@ -0,0 +1,10 @@
+#
+# Copyright (c) 2023 Renesas Electronics Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+#
+CONFIG_HEAP_MEM_POOL_SIZE=4096
+CONFIG_LOG_MODE_MINIMAL=y
+CONFIG_MIPI_DBI_LOG_LEVEL_DBG=y
```

Comment:
Are the log settings needed for customers running the sample? Further, does the heap size need adjustment? It seems like the default (larger) heap size would work ok.